src/bokeh/sphinxext/_internal/bokeh_model.py

```diff
@@ -90,7 +90,7 @@ def run(self):
         m = py_sig_re.match(sig)
         if m is None:
             raise SphinxError(f"Unable to parse signature for bokeh-model: {sig!r}")
-        name_prefix, model_name, arglist, retann = m.groups()
+        _, model_name, __, ___ = m.groups()
```

Comment:
I'm not sure if I find this as an improvement. I think:
```py
_name_prefix, model_name, _arglist, _retann = m.groups()
```
would have been better in this and related cases. Though the optimal approach would be to switch to named groups.

Reply:
I prefer this and in any case it's only in ~three places in the sphinxext code so I'm not inclined to change it.

Reply:
[95a949a](https://github.com/bokeh/bokeh/pull/14661/commits/95a949a98e6cc82929c01c439b6a103dddd20e02)